src/phone/dto/update-phone.dto.ts

```diff
@@ -1,9 +1,10 @@
 /* eslint-disable prettier/prettier */
 import { PartialType } from '@nestjs/mapped-types';
 import { CreatePhoneDto } from './create-phone.dto';
+import { UserInterface } from '../../user/interfaces/user.interface';
 
 export class UpdatePhoneDto extends PartialType(CreatePhoneDto) {
   id?: string;
-  userId: string;
+  user: UserInterface;
   phoneNumber: string;
 }
```

Comment:
Tą klasę można zostawić pustą jako, że dziedziczy po PartialType(CreatePhoneDto), Darek to tłumaczył